controllers/userController.js

```diff
@@ -0,0 +1,31 @@
+const { catchAsync } = require('../utils/error');
+const userService = require('../services/userService');
+
+const signUp = catchAsync(async (req, res) => {
+  const { first_name, last_name, email, password } = req.body;
+
+  if (!first_name || !last_name || !email || !password) {
+    const error = new Error('KEY ERROR');
+    error.statusCode = 400;
+    throw error;
+  }
+  await userService.signUp(first_name, last_name, email, password);
+
+  res.status(201).json({ message: 'CREATED USER' });
+});
+
+const signIn = catchAsync(async (req, res) => {
+  const { email, password } = req.body;
+
+  try {
+    const accessToken = await userService.signIn(email, password);
+    res.status(200).json({ accessToken: accessToken });
```

Comment:
```suggestion
    res.status(200).json({ accessToken });
```
property name과 value에 들어가는 변수의 name이 같다면 위와같이 쓰는 것이 보다 가독성에 좋습니다.